python-packages/order_utils/CHANGELOG.md

```diff
@@ -10,3 +10,7 @@
 -   Deprecated methods `encode_erc20_asset_data()` and `encode_erc721_asset_data()`, in favor of new methods `encode_erc20()` and `encode_erc721()`. The old methods return a string, which is less than convenient for building orders using the provided `Order` type, which expects asset data to be `bytes`. The new methods return `bytes`.
 -   Expanded documentation.
 -   Stopped using deprecated web3.py interface `contract.call()` in favor of `contract.functions.X.call()`. This provides compatibility with the upcoming 5.x release of web3.py, and it also eliminates some runtime warning messages.
+
+## 3.0.0 - TBD
+
+-   Major breaking changes: removal of definitions for Order, OrderInfo, order_to_jsdict, jsdict_to_order, all of which have been moved to contract_wrappers.exchange.types; removal of signature validation.
```

Comment:
Nit: Can we invert the ordering of the CHANGELOG so that the latest version is at the top?

Reply:
https://github.com/0xProject/0x-monorepo/pull/1919/commits/2a5a27f51df1c6ec0ea864e86fa462e3e00a61bf